calico_node/filesystem/startup.py

```diff
@@ -262,7 +274,7 @@ def main():
     # Create default pools if required
     if not ipv4_pools:
         client.add_ip_pool(4, DEFAULT_IPV4_POOL)
-    if not ipv6_pools and ipv6_enabled():
+    if not ipv6_pools and os.path.exists('/proc/sys/net/ipv6'):
```

Comment:
I prefer either having the ipv6_enabled() method or at least a comment to demystify the path exists statement.